modules/search/class-jetpack-instant-search.php

```diff
@@ -8,6 +8,7 @@
 
 use Automattic\Jetpack\Connection\Client;
 use Automattic\Jetpack\Constants;
+require_once __DIR__ . '/class.jetpack-search-helpers.php';
```

Comment:
Do we need this here, even though the file is already required by the base class?

Reply:
Ah, good point; this require is unnecessary.